docs/topics/db/transactions.txt

```diff
@@ -622,9 +622,9 @@ Handling exceptions within PostgreSQL transactions
 Inside a transaction, when a call to a PostgreSQL cursor raises an exception
 (typically ``IntegrityError``), all subsequent SQL in the same transaction
 will fail with the error "current transaction is aborted, queries ignored
-until end of transaction block". While simple use of ``save()`` is unlikely
-to raise an exception in PostgreSQL, there are more advanced usage patterns
-which might, such as saving objects with unique fields, saving using the
+until end of transaction block". While the use of ``save()`` is unlikely to
```

Comment:
Maybe "basic use of save()"? `save()` is still used in the following "more advanced usage patterns"